.github/PULL_REQUEST_TEMPLATE/frontend_template.md

```diff
@@ -1,46 +1,46 @@
 <!--- Provide a general summary of your changes in the Title above -->
 <!-- This is a template, feel free to remove unused sections or add information you think it's needed -->
 
+### Linked PRs
+<!-- (Optional) Please list any related PRs here:
+- PR #123 (Title and link)
+- PR #456 (Title and link)
+-->
```

Comment:
👍 The list format ensures that the PR title is expanded and the state of the PR (pending/merged) is directly visible.